tests/test-content/js/custom/custom.js

```diff
@@ -0,0 +1,12 @@
+// Copyright 2018 Endless Mobile, Inc.
+
+/* exported Custom */
+
+const Gtk = imports.gi.Gtk;
+
+const Module = imports.app.interfaces.module;
```

Comment:
Now that I think about it, adding the ability to ship JS in the app makes `imports.app` a really confusing name. Maybe we can rename it to `imports.framework` or something?

Reply:
I don't foresee that, because the only code that could possibly have used the old name is inside eos-knowledge-lib.